test.js

```diff
@@ -100,4 +100,33 @@ describe('gce-images', function () {
       });
     });
   });
-});
\ No newline at end of file
+
+  describe('_sortNewestFirst', function () {
+    describe('when imageA.creationTimestamp is newer than imageB.creationTimestamp', function () {
+      it('should return 1', function () {
```

Comment:
Oops, can you change that to shift and then revert to the original behavior of the comparator?